index.md

```diff
@@ -31,57 +37,52 @@ layout: default
-| [lerna-log](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-log.md) | Lerna Log library provides logging related features like below. <br> - Custom log converters for [Logback](http://logback.qos.ch/) |
-| [lerna-management](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-management.md) | Lerna Management library provides management related features like below. <br> - SNMP metrics provider for [Kamon](https://kamon.io/) |
-| [lerna-testkit](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-testkit.md) | Lerna TestKit library provides testkits for <br> - [Akka Classic](https://doc.akka.io/docs/akka/current/index-classic.html) <br> - [Airframe](https://wvlet.org/airframe/) <br> - [WireMock](http://wiremock.org/) |
-| [lerna-util](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-util.md) | Lerna Util library provides some utilities like below. <br> - Encryption <br> - Typed Equals <br> - Security <br> - Date and Time |
-| [lerna-util-akka](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-util-akka.md) | Lerna Util Akka library provides some utilities related Akka Classic. |
-| [lerna-util-sequence](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-util-sequence.md) | Lerna Util Sequence library provides an ID generator. | 
-| [lerna-validation](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-validation.md) | Lerna Validation library provides custom validators for [Accord](http://wix.github.io/accord/). |
-| [lerna-wart-core](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-wart-core.md) | Lerna Wart Core library provides custom warts for [WartRemover](https://www.wartremover.org/) |
-| [nablarch-fw-batch-parallelizable](https://github.com/lerna-stack/nablarch-fw-batch-parallelizable) | a library for parallelizing batch processing with [Nablarch Batch Framework](https://nablarch.github.io/docs/LATEST/doc/application_framework/application_framework/batch/index.html) |
+| [application template](https://github.com/lerna-stack/lerna.g8) | a template for creating a new project |
+| [terraform scripts](https://github.com/lerna-stack/lerna-terraform) | scripts to build the environment |
+| [learning content](https://github.com/lerna-stack/lerna-handson) | hands-on content learning Akka and Scala with Lerna |
+| [system design documents](https://github.com/lerna-stack/lerna-design) | documentation for developing with Lerna |
+| [application libraries](#application-libraries) | libraries for developing applications that run on Lerna Stack ( [API Doc](https://lerna-stack.github.io/scaladoc/index.html) ) |
+| [sample application](https://github.com/lerna-stack/lerna-sample-payment-app)  | sample applications for developing with Lerna application libraries |
```

Comment:
@yugolf 

[nits] It may better to write "A sample application" in the singular.

Reply:
I changed the sentences as you pointed out.